src/MOI/MOI_wrapper.jl

```diff
@@ -816,10 +849,13 @@ function MOI.add_variable(model::Optimizer)
         [-Inf],#_dbdl::Vector{Float64},
         [Inf],#_dbdu::Vector{Float64}
     )
+
+    push!(model.nlp_variable_info, NLPVariableInfo())
```

Comment:
remove...

Reply:
If I add another function MOI.add_variable with the same arguments for NLP case, MOI will always use the old function. Thus, it's the only solution I found to solve this issue